scripts/dashboard/install-e2e-test-deps.sh

```diff
@@ -11,7 +11,7 @@ if grep -q  debian /etc/*-release; then
     curl -fsSL https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
     sudo apt-get update
     sudo apt-get install -y google-chrome-stable
-    sudo apt-get install -y python-requests python-openssl python-jinja2 python-jwt python-scipy
+    sudo apt-get install -y python-requests python3-requests python-openssl python3-openssl python-jinja2 python3-jinja2 python-jwt python3-jwt python-scipy python3-scipy
```

Comment:
could you take this opportunity to remove these python2 dependencies? as we are now pure python3.

Reply:
But don't we still need it for the nautilus nightly job and for PRs targeting the nautilus branch?

Reply:
ahh, very true! i forgot this. thanks for reminding me.